packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/gantt/PapayaGanttControllerIntegrationTests.java

```diff
@@ -135,14 +142,15 @@ public void givenGanttRepresentationWhenWeSubscribeToItsEventThenTheRepresentati
     }
 
     @Test
-    @DisplayName("Given a gantt representation, when we create a task, then the new task is visible")
+    @DisplayName("Given a gantt representation, when we create/edit a task, then the new task is created/edited")
     @Sql(scripts = {"/scripts/papaya.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
     @Sql(scripts = {"/scripts/cleanup.sql"}, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
-    public void givenGanttRepresentationWhenWeCreateTaskThenTheNewTaskIsVisible() {
+    public void givenGanttRepresentationWhenWeCreateEditTaskThenTheTaskIsAsExpected() {
```

Comment:
I don't think this new test will correct the flaky test coverage on `collaborative-gantt`.
`RenameGanttEventHandler` and `DropTaskEventHandler` are not tested so depending on the verification order of the canHandle, code coverage may still change

Reply:
yes indeed but this PR increase the coverage 
* for `sirius-components-collaborative-gantt` from 75 to 82%
* for `sirius-components-gantt` from 83 to 91%

It is a good step, so to avoid the future instable test result just set the new references to the max minus 1%
* 81% for `sirius-components-collaborative-gantt`
* 90% for `sirius-components-gantt`

We don't have to stand in each other's way!